bicycleparameters/cycle_app.py

```diff
@@ -0,0 +1,313 @@
+import dash
+import dash_core_components as dcc
+import dash_html_components as html
+import dash_table as tbl
+from dash.dependencies import Input, Output, State
+
+import bicycleparameters as bp
```

Comment:
This can be a relative import. I think this is what you want:

```python
from . import bicycleparameters as bp
```

Reply:
That's what I thought as well. However, when I use it, I receive the error message below. I looked around online and couldn't find a direct solution, but I did see some things mentioning sys module. Wasn't really sure how to proceed.

````
File "c:/Users/xyzasdf/Desktop/bike-lab/BicycleParameters/bicycleparameters/cycle_app.py", line 7, in <module>
    from . import bicycleparameters as bp
ImportError: attempted relative import with no known parent package

Reply:
I've been at this for hours and still no use. I've been trying to do other sorts of imports from within the bicycleparameters directory and absolutely nothing is working. They all give similar "no parent package" errors. Here is one call made from within the interpreter which may shed some light, although I don't personally know entirely how to interpret it.

```
C:\Users\xyzasdf\Desktop\bike-lab\BicycleParameters\bicycleparameters>__init__.py
Traceback (most recent call last):
  File "C:\Users\xyzasdf\Desktop\bike-lab\BicycleParameters\bicycleparameters\__init__.py", line 3, in <module>
    from .main import Bicycle
ImportError: attempted relative import with no known parent package
```

Reply:
Actually I think it just needs to be what you have: `import bicycleparameters as bp`. This file shouldn't use relative imports because it isn't imported.

Reply:
Sorry, I may have wasted a bunch of your time with the first statement. I haven't packaged in this precise way before so I assumed wrong.

Reply:
You may have to run it from the main directory though:

```
$ pwd
/path/to/BicycleParameters
$ python bicycleparameters/cycle_app.py
```